addons/gut/test.gd

```diff
@@ -237,6 +237,18 @@ func assert_ne(got, not_expected, text=""):
 			_fail(disp)
 		else:
 			_pass(disp)
+
+# ------------------------------------------------------------------------------
+# Asserts that the expected value roughly equals the value got.
+# ------------------------------------------------------------------------------
+func assert_roughly_eq(got, expected, error_interval, text=""):
```

Comment:
Please, add `assert_roughly_nq` method.

Reply:
I like the idea of an inverse of this assert.  I requested the name change to `assert_almost_eq`.  So the inverse of this would be `assert_almost_ne`.  I'm not sure I like that but I can't think of a better name.

If you can get this into the PR that would be great.  If not then I'll add it to the issue I'm about to add that does the same thing for `assert_between`.

Reply:
Yes, I liked your name suggested.